.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,36 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Is everything committed in the `.idea/` folder here useful for other devs? Or is it configuration unique to your setup? I'm wondering whether we should add `.idea` to the `.gitignore` file and not commit all this.